cluster-scope/overlays/nerc-ocp-edu/clusterrolebindings/autopilot-service-acc-node-reader.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: autopilot-service-acc-node-reader
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: nerc-node-reader
+subjects:
+  - kind: ServiceAccount
+    name: autopilot-service-acc
+    namespace: autopilot-dashboard-f3dc9e
```

Comment:
Not sure if it matters, but I don't see where this namespace is created. This seems to be a coldfront generated name.

The namespace created by the auotpilot [bundle](https://github.com/OCP-on-NERC/nerc-ocp-config/blob/main/cluster-scope/base/core/namespaces/autopilot/namespace.yaml) is `autopilot`.

Reply:
I've removed them from the cluster and this overlay for now. We can add them back later if they're still needed on this new edu/academic cluster.